src/main/java/com/newrelic/telemetry/opentelemetry/export/NewRelicMetricExporter.java

```diff
@@ -16,14 +16,35 @@
 import java.net.URI;
 import java.util.Collection;
 
+/**
+ * The NewRelicMetricExporter takes a collection of MetricData objects, converts them into a New
+ * Relic MetricBatch instance and then sends it to the New Relic metric ingest API via a
+ * TelemetryClient.
+ *
+ * <p>This is a "push-based" exporter, and so will need to be hooked up with a {@link
+ * io.opentelemetry.sdk.metrics.export.IntervalMetricReader} in order
+ *
+ * @since 0.3.0
+ */
 public class NewRelicMetricExporter implements MetricExporter {
 
   private final Attributes commonAttributes;
   private final TelemetryClient telemetryClient;
   private final TimeTracker timeTracker;
   private final MetricPointAdapter metricPointAdapter;
 
-  public NewRelicMetricExporter(
+  /**
+   * Create a metric exporter with the given components.
+   *
+   * @param telemetryClient A {@link TelemetryClient} instance
+   * @param serviceAttributes A set of attributes to be appended to all metrics generated by this
+   *     exporter.
+   * @param timeTracker A {@link TimeTracker} used to compute delta values from cumulative values,
+   *     where necessary.
+   * @param metricPointAdapter The {@link MetricPointAdapter} that converts OpenTelemetry Point
+   *     instances into New Relic {@link Metric}s.
+   */
+  NewRelicMetricExporter(
```

Comment:
Did you intend to change the visibility of this?

Reply:
yeah, there's not need for this constructor to be public.